lib/rules/declaration-block-no-duplicate-properties/__tests__/index.js

```diff
@@ -48,6 +52,10 @@ testRule(rule, {
   ],
 
   reject: [
+    {
+      code: "color: pink; color: orange",
```

Comment:
If this is for `style=""` attributes? If so, how does it differ from the `syntax: html` tests below?